docs/source/user_guide/poweruser/daft-launcher.rst

```diff
@@ -0,0 +1,12 @@
+.. _daft_launcher:
+
+Daft Launcher
+=====================
+
+Getting started with running daft at a distributed scale can be tough.
+Therefore, we've created a simple, open-sourced launcher to help you get started.
+
+The tool, `daft-launcher <https://github.com/Eventual-Inc/daft-launcher>`_, is open-sourced and available to be run today!
+To install it, you can run ``pip install daft-launcher`` (or using ``uv``, you can run ``uv pip install daft-launcher``).
```

Comment:
Just double checking here, on your daft launcher docs the uv command has daft-launcher wrapped in quotes - `uv pip install "daft-launcher"`. does it make a difference?

Reply:
No, it shouldn't make a difference. However, that definitely can be confusing to end-users... I will change it.

Reply:
I'll deprecate the Daft Launcher docs (and add a link pointing towards the Daft main docs instead).